src/pages/login.tsx

```diff
@@ -22,6 +22,9 @@ export default function LoginForm() {
             router.push('/')
         } else {
             setMsg(` ${data.error}`);
+            setTimeout(() => {
```

Comment:
why is this? It looks like a hack that you are fixing something on the wrong place,